packages/web-components/src/components/masthead/masthead-contact.ts

```diff
@@ -45,6 +47,18 @@ class DDSMastheadContact extends DDSMastheadProfile {
     `;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('expanded')) {
+      if (!this.expanded) {
```

Comment:
Given the outer condition is always `true`, I believe that _may_ result in an unnecessary call to `init()` on the first render here, since `this.expanded` is `flase` on first render. I guess it depends on if the `mastheadContainer.contactModuleApp` is ready yet. Woulud it make sense to change the condition here to `changedProperties.get('expanded') === true && !this.expanded` ?

Reply:
As I noted on the above comment, that value doesn't always eval to `true`. It will evaluate to false if the value of `this.expanded` is not what caused this update lifecycle.